Makefile

```
@@ -82,10 +83,29 @@ test_c_mem: $(SO_FILE) $(TEST_FILES)
 	export REPORT_EXIT_STATUS=1; \
 	export TEST_PHP_SRCDIR=$(BUILD_DIR); \
 	export USE_TRACKED_ALLOC=1; \
-	\
-	$(MAKE) -C $(BUILD_DIR) CFLAGS="-g" clean all; \
 	php -n -d 'memory_limit=-1' $$TEST_PHP_SRCDIR/run-tests.php -n -p $$(which php) -d extension=$(SO_FILE) -q --show-all -m $(TESTS)
 
+test_c2php: $(SO_FILE) $(INIT_HOOK_TEST_FILES)
+	( \
+	set -xe; \
+	export DD_TRACE_CLI_ENABLED=1; \
+	export USE_ZEND_ALLOC=0; \
+	export ZEND_DONT_UNLOAD_MODULES=1; \
+	export USE_TRACKED_ALLOC=1; \
+	valgrind -q --tool=memcheck --trace-children=yes --vex-iropt-register-updates=allregs-at-mem-access php -n -d extension=$(SO_FILE) -d ddtrace.request_init_hook=$$(pwd)/bridge/dd_wrap_autoloader.php tests/C2PHP/get_context_distributed_tracing_test.phpt; \
```

Should this be pointed to `$(INIT_HOOK_TEST_FILES)`?

```suggestion
	valgrind -q --tool=memcheck --trace-children=yes --vex-iropt-register-updates=allregs-at-mem-access php -n -d extension=$(SO_FILE) -d ddtrace.request_init_hook=$$(pwd)/bridge/dd_wrap_autoloader.php $(INIT_HOOK_TEST_FILES); \
```

---

good catch!